3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb

```diff
@@ -250,6 +279,7 @@
     "            keys=[\"image\"], a_min=-175, a_max=250, b_min=0.0, b_max=1.0, clip=True\n",
     "        ),\n",
     "        CropForegroundd(keys=[\"image\", \"label\"], source_key=\"image\"),\n",
+    "        EnsureTyped(keys=[\"image\", \"label\"], device=device, track_meta=True),\n",
```

Comment:
I think maybe you didn't get my idea in the previous comment, I mean you can change it to:
```py
EnsureTyped(keys=["image", "label"], device=device, track_meta=get_track_meta())
```
The `get_track_meta` API depends on `set_track_meta(True/False)`, so you use `set_track_meta(True)` before & after training, use `set_track_meta(False)` during the training, the validation transform can work well in all the cases, right?

Thanks.

Reply:
Sorry I may understand your change, it looks good to me.

Thanks.